<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC
    "-//W3C//DTD XHTML 1.1 plus MathML 2.0 plus SVG 1.1//EN"
    "http://www.w3.org/2002/04/xhtml-math-svg/xhtml-math-svg.dtd">
<html xml:lang='en' xmlns:svg='http://www.w3.org/2000/svg' xmlns='http://www.w3.org/1999/xhtml'>
<head><meta content='application/xhtml+xml;charset=utf-8' http-equiv='Content-type' /><title>webMetAl FAQ</title></head>
<body>
<h1 id='webmetal_faq'>webMetAl FAQ</h1>

<ul>
<li>What is MetAl?</li>
</ul>

<p>MetAl is a set of metrics for measuring the distance between alternative multiple sequence alignments. Four metrics are available, &#8220;SSP&#8221;, &#8220;seq&#8221;, &#8220;pos&#8221; and &#8220;evol&#8221;. The &#8220;evol&#8221; metric requires a phylogenetic tree.</p>

<p>For more information, see our paper <a href='http://dx.doi.org/10.1093/bioinformatics/btr701'>Measuring the distance between mutliple sequence alignments</a>.</p>

<ul>
<li>What is a metric?</li>
</ul>

<p>A metric defines a distance between elements of a set; in the case of MetAl the metrics are over the set of all possible alignments of a given set of sequences. One important feature is that the results from MetAl can be treated as distances.</p>

<ul>
<li>What is webMetAl?</li>
</ul>

<p>WebMetAl implements the MetAl metrics in pure JavaScript, so it can run in your web browser. You can then visualise the distances at the level of individual sequences and individual characters.</p>
</body></html>
